server/routes/routerRequestHandlers.test.ts

```diff
@@ -698,6 +705,13 @@ describe('routerRequestHandlers', () => {
       req.session.affectAbilityToWorkForm = {} as AffectAbilityToWorkForm
       req.session.reasonsNotToGetWorkForm = {} as ReasonsNotToGetWorkForm
       req.session.workInterestTypesForm = {} as WorkInterestTypesForm
+      req.session.workInterestRolesForm = {} as WorkInterestRolesForm
```

Comment:
Our router request handler that clears out all the induction related forms from the session had a few missing (see change to the function implementation below 👇 )